ch9.md

```diff
@@ -311,14 +311,14 @@ If we want to keep the Observer and Observable separate:
 
 // producer:
 
-var a = Rx.Observable.create( function onObserver(observer){
+var a = Rx.Observable.create( function onObserve(observer){
 	setInterval( function everySecond(){
-		a.onNext( Math.random() );
+		observer.next( Math.random() );
 	}, 1000 );
 } );
 ```
 
-In this snippet, `a` is the Observable, and unsurprisingly, the separate observer is called `observer`; it's able to "observe" some events (like our `setInterval(..)` loop) and publish events to the `a` observable stream.
+In this snippet, `a` is the Observable, and unsurprisingly, the separate observer is called `observer`; it's able to "observe" some events (like our `setInterval(..)` loop) and ~~publish events to the `a` observable stream~~.
```

Comment:
I'm even more confused. These two phrases from you seem contradictory:

* "It's the opposite of "observer pushes data to the observable"
* "the act of pushing data into a Subject is the Observer part of the Subject"

So which is it? Does the observer push data to the observable, or does the observable push data to the observer?